.azure-pipelines/windows.sh

```diff
@@ -114,11 +111,22 @@ cd "${DMD_DIR}/../druntime"
 ################################################################################
 cd "${DMD_DIR}/test"
 
-# WORKAROUND: Copy the built Phobos library in the path
-# REASON: LIB argument doesn't seem to work
-cp "${DMD_DIR}/../phobos/$LIBNAME" .
+if [ "$MODEL" == "32" ] ; then
+    # WORKAROUND: Make Optlink use freshly built Phobos, not the host compiler's.
+    # Optlink apparently prefers LIB in sc.ini over the LIB env variable (and
+    # `-conf=` for DMD apparently doesn't prevent that).
+    # There's apparently no sane way to specify a libdir for Optlink in the DMD
+    # cmdline, so remove the sc.ini file and set the DFLAGS and LIB env variables
+    # manually for the host compiler. These 2 variables are adapted for the
+    # actual tests with the tested compiler (by run.d).
+    HOST_DMD_DIR="$(cygpath -w "$DMD_DIR/tools/dmd2")"
+    rm "$HOST_DMD_DIR/windows/bin/sc.ini"
```

Comment:
Thanks for tackling this.

Instead of breaking the host compiler installation (in case someone uses the script outside of the CI), placing an empty sc.ini into the current directory should also work, because that's preferred by optlink.

Reply:
A quick local test shows that's not the case. DMD prefers that sc.ini in the current dir, but Optlink isn't, regardless of whether it's completely empty or contains an empty `[Environment]` section, and regardless of whether Optlink is invoked via DMD or directly.

Reply:
Ok. I was looking at this comment: https://github.com/DigitalMars/optlink/blob/87a85b0413d94fb8017656004a03f7fb0aa3a649/common/iniproc.asm#L52

I guess no one uses the CI build scripts locally.